WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController/NotificationsTableHeaderView.swift

```diff
@@ -56,18 +62,41 @@ final class NotificationsTableHeaderView: UITableViewHeaderFooterView {
         self.contentConfiguration = config
     }
 
+    private func update(style: Style) {
+        if var config = self.contentConfiguration as? UIListContentConfiguration {
+            switch style {
+            case .leading:
+                config.directionalLayoutMargins = Appearance.layoutMarginsLeading
+            case .subsequent:
+                config.directionalLayoutMargins = Appearance.layoutMarginsSubsequent
+            }
+            self.contentConfiguration = config
+        }
+    }
+
     // MARK: - Constants
 
     private enum Appearance {
         static let backgroundColor = UIColor.systemBackground
         static let textColor = UIColor.DS.Foreground.primary ?? .text
         static let textFont = UIFont.DS.font(.bodyLarge(.emphasized))
-        static let layoutMargins = NSDirectionalEdgeInsets(
+        static let layoutMarginsLeading = NSDirectionalEdgeInsets(
             top: Length.Padding.double,
             leading: Length.Padding.double,
-            bottom: Length.Padding.double,
+            bottom: Length.Padding.half,
+            trailing: Length.Padding.double
+        )
+        static let layoutMarginsSubsequent = NSDirectionalEdgeInsets(
+            top: Length.Padding.double + Length.Padding.split,
+            leading: Length.Padding.double,
+            bottom: Length.Padding.half,
             trailing: Length.Padding.double
         )
     }
 
+    enum Style {
+        case leading
+        case subsequent
+    }
+
```

Comment:
Is it possible to name this to something more elaborate? I couldn't understand what it is without reading its usage. Also could you move this enum to the top of the class right below the class signature (unless you're against it)? For public nested types, it is common to list them on the top so when a person views the file, all relevant types are at the top rather than being scattered through the file.

```final class NotificationsTableHeaderView: UITableViewHeaderFooterView {
    enum ... {
        ...
    }
```

Reply:
Totally makes sense! Will try to find more accurate naming.